app/assets/javascripts/reducers/articles.js

```diff
@@ -8,6 +8,14 @@ const initialState = {
   sortKey: 'character_sum'
 };
 
+const SORT_DESCENDING = {
+  rating_num: true,
+  title: true,
```

Comment:
To match the current behavior, just character_sum and view_count should be here.